src/_codux/boards/donation-card/donation-card.board.tsx

```diff
@@ -0,0 +1,25 @@
+import { createBoard } from '@wixc3/react-board';
+import DonationCard from '../../../components/DonationCard/DonationCard';
+
+const donationCardData = {
+   id: 1,
+   title: 'Save the Whales',
+   text: 'Help us save endangered whale species around the world. Your contributions make a difference.',
+   target: 5000,
+};
+
+export default createBoard({
+   name: 'DonationCard',
+   Board: () => (
+      <DonationCard
+         title={donationCardData.title}
+         text={donationCardData.text}
+         target={donationCardData.target}
+      />
+   ),
+   isSnippet: true,
+   environmentProps: {
+      canvasWidth: 1161,
+      canvasHeight: 149,
```

Comment:
Please use more logical sizes, these numbers feel too random. You can omit them too.